### DISCUSSION TITLE
Welcome to libgd Discussions!

### DISCUSSION THREAD

**Discussion by pierrejoye** [libgd/libgd#727] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhouming1107** [comment]
Hello:
  when i  ./configure , it shows :Support for gd/gd2 images:  no,.I need it show yes .What do I need to do. Thanks

**Reply by cmb69** [reply]
> I need it show yes .What do I need to do.

See `./configure --help | grep gd`. I suggest to migrate away from these formats, though.

Also note that support questions should be filed as [issues](https://github.com/libgd/libgd/issues), if at all. There are plenty of support channels on the internet which are better suited.

**Comment by pierrejoye** [comment]
Discussions are fine for that. So issues are about issues or features
requests  :)

**Comment by wglb** [comment]
I'm trying to use this library on FreeBSD. When I link with a test program, I get
`-*- mode: compilation; default-directory: "/usr/home/wgl/quicklisp/dists/quicklisp/software/cl-gd-20201220-git/" -*-
Compilation started at Fri Jul  8 15:06:30

make 
cc -I/usr/local/include -fPIC -c cl-gd-glue.c
ld -shared -lgd -lz -lpng -ljpeg -lfreetype -lm -lc cl-gd-glue.o -o cl-gd-glue.so -L/usr/local/lib
ld: error: unable to find library -lz
ld: error: unable to find library -lm
ld: error: unable to find library -lc
*** Error code 1
`

Which libraries are -lz, -lm, and -lc?

**Reply by cmb69** [reply]
> Which libraries are -lz, -lm, and -lc?

zlib, libm(ath) and libc. These are common on Linux; don't know much about FreeBSD.
